javascript/js-in-the-real-world/forms.md

```diff
@@ -1,42 +1,56 @@
 ### Introduction
-Forms are a crucial part of most websites. Almost every major site has sign-up forms, contact forms, search forms and more!  Luckily HTML5 and JavaScript have some handy built-in methods.
 
-- [This tutorial](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation) covers pretty much everything you need to know about HTML form validation.
-- For Reference, [this document](https://www.w3schools.com/js/js_validation_api.asp) covers the JavaScript validation API in a more concise format.  These functions were explained in the previous article.  Typically, with HTML forms, the inputs are validated upon form submission, but you can use these functions to check validity whenever you like (such as when a user clicks or tabs out of a specific input field).
+Forms are a crucial part of most websites. Almost every major site has sign-up forms, contact forms, search forms and more!  Luckily HTML5 and JavaScript have some handy built-in methods. You've already learned about validation with HTML and styling validations with CSS in our [Form Validations](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css/lessons/form-validation) lesson in the Intermediate HTML and CSS course.
+
+In this lesson, we'll cover the Constraint Validation API: a way to validate forms on the frontend with JavaScript.
 
 ### Learning Outcomes
 By the end of this lesson, you should be able to do the following:
 
 - Understand the importance of validation in HTML forms.
-- Add validation attributes in HTML forms.
 - Understand Constraint Validation API for more control over form validation.
-- Add custom styling to validation errors.
 - Add validation using only JavaScript.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. [This tutorial on Form Validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_forms_using_javascript) covers pretty much everything you need to know about HTML form validation. It starts off with HTML attributes, but keep on reading the doc covers the Constraint Validation API. 
```

Comment:
Believe just one last request. We should remove mention of HTML form validation here. Maybe something along the lines of:

```suggestion
1. [This tutorial on Form Validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_forms_using_javascript) covers how we can use JavaScript to validate forms, including the constraint validation API.
```